src/assets/fake_data.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
For the sake of consistency, would you mind moving this data file to the `src/data/` directory?

Reply:
yep yep!